prime-router/src/main/kotlin/azure/BlobAccess.kt

```diff
@@ -19,9 +22,11 @@ import java.net.URL
 import java.net.URLDecoder
 import java.nio.charset.Charset
 import java.security.MessageDigest
+import java.time.Duration
 
 const val defaultBlobContainerName = "reports"
 const val defaultBlobDownloadRetryCount = 5
+const val listBlobTimeoutSeconds: Long = 30
```

Comment:
If this operation could actually take 30 seconds I think we should consider making it async.